tests_e2e/orchestrator/scripts/install-tools

```diff
@@ -47,25 +71,52 @@ if [[ -e ~/bin/python3 ]]; then
 fi
 ln -s ~/bin/pypy*/bin/pypy3.7 ~/bin/python3
 
-# Note that we place $HOME/bin at the front of PATH, so the tools in ~/bin (including python) will
+echo "Installing Python modules"
+python3 -m ensurepip
+python3 -mpip install -U pip wheel
+python3 -mpip install distro
+
+echo "Installing Agent modules to ~/bin"
+unzip.py "$agent_package" ~/bin/WALinuxAgent
+unzip.py ~/bin/WALinuxAgent/bin/WALinuxAgent-*.egg ~/bin/WALinuxAgent/bin/WALinuxAgent.egg
+mv ~/bin/WALinuxAgent/bin/WALinuxAgent.egg/azurelinuxagent ~/bin
+rm -rf ~/bin/WALinuxAgent
+
+#
+# Create ~/bin/agent-env to set PATH and PYTHONPATH.
+#
+# We add $HOME/bin to the front of PATH, so tools in ~/bin (including python) will
 # take precedence over system tools with the same name.
+#
+# We set PYTHONPATH to include the location of the agent modules installed on the VM image and also
+# the test modules we copied to ~/bin.
+#
+#
+echo "Creating ~/bin/agent-env to set PATH and PYTHONPATH"
```

Comment:
I moved this to its own file (agent-env) instead of adding it directly to .bash_profile. It turns out that in some distros (e.g. Mariner) .bash_profile references undefined variables, and that was adding some really weird error messages to our logs.

I'm still adding agent-env to .bash_profile to help with interactive debugging sessions. If you ssh to a test machine, just keep in mind PATH and PYTHONPATH are setup to the same values as in the test runs (e.g. 'python3' will invoke Pypy, instead of the Python installed on the image)